lib/modules/manager/flux/index.ts

```diff
@@ -7,8 +8,8 @@ import { GitlabTagsDatasource } from '../../datasource/gitlab-tags';
 import { HelmDatasource } from '../../datasource/helm';
 import { systemManifestRegex } from './common';
 
-export { extractAllPackageFiles, extractPackageFile } from './extract';
 export { updateArtifacts } from './artifacts';
+export { extractAllPackageFiles, extractPackageFile } from './extract';
```

Comment:
please revert this, you probably need some extensions for your vscode, so it will follow our code style

checkout our recommend extensions  inside `.vscode` folder

Reply:
I'm using the vscode devcontainer and also ran `yarn run lint-fix` before I commit this. Shouldn't both cases here cause that not to happen?